docs/explanations/pro_token_and_machine_usage.rst

```diff
@@ -0,0 +1,71 @@
+.. _pro_token_and_machine_usage:
+
+Machine counts and token usage for Ubuntu Pro
+*********************************************
+
+Ubuntu Pro tokens are used to authenticate your machine and enable access to the services you are entitled to. tokens are unique and specific to your account, so you should treat them as a secret (as you would with a password).
+
+Since tokens are attached to your account and not your machine, you can re-use tokens, even multiple times per day. In this page, we'll discuss how tokens work and how machines are counted against your subscription.
+
+Pro tokens
+----------
+
+- Obtaining tokens:
+    After subscribing to Ubuntu Pro, you can access your tokens through the `Ubuntu Pro Dashboard <Pro_dashboard_>`_. Each token can be used to attach multiple machines depending on the subscription.
```

Comment:
I think we should emphasize the 1:1 relationship between subscriptions and tokens.

Maybe something like this:
```suggestion
    After subscribing to Ubuntu Pro, you can access your subscriptions through the `Ubuntu Pro Dashboard <Pro_dashboard_>`_. Each subscription includes a token that can be used to attach as many machines as the subscription allows.
```